File: 0190-reverse-bits/0190-reverse-bits.java
import java.util.*;
public class Solution {
    public int reverseBits(int n) {
        int count=31;
        int sum=0;
        for(int i=0;i<31;i++)
        {
            int a=n%2;
            sum+=(int)a*(int)(Math.pow(2,count));
            count-=1;
            n=n/2;
        }
        return sum;
    }
}

File: 0190-reverse-bits/0190-reverse-bits.py
class Solution:
    # @param n, an integer
    # @return an integer
    def reverseBits(self, n):
        sum = 0
        count = 31
        while n > 0:
            bit = n % 2
            sum = sum + (bit * (2 ** count))
            n = n // 2
            count -= 1 
        return sum
